vms/avm/vm.go

```diff
@@ -64,6 +66,13 @@ var (
 	_ vertex.DAGVM = &VM{}
 )
 
+const (
+	utxoDbPrefix byte = iota
+	txDbPrefix
+	statusDbPrefix
+	idDbPrefix
+)
```

Comment:
It seems like in other places strings are used. Let's try to keep it them same everywhere.

Reply:
This avoids unnecessary byte copying but I'm alright with using strings, too

Reply:
I'll revert this in that case.